src/02-common-principles.md

```diff
@@ -644,6 +540,150 @@ for more information.
+
+1.  A metadata file MUST NOT have a filename that would be otherwise applicable
+    to some data file based on rules 2.2 and 2.3 but is made inapplicable based on its
+    location in the directory structure as per rule 2.1.
+
+1.  There MUST NOT be multiple metadata files applicable to a data file at one level
+    of the directory hierarchy.
+
+1.  If multiple metadata files are applicable to a given data file:
+
+    1.  For [tabular files](#tabular-files) and other simple metadata files
+        (for instance, [`bvec` / `bval` files for diffusion MRI](#04-modality-specific-files/01-magnetic-resonance-imaging#required-gradient-orientation-information),
+        only the file lowest in the filesystem hierarchy SHALL be treated as being
+        associated with that data file.
+
+    1.  For [JSON files](#key-value-files-dictionaries):
+
+        1.  Files are loaded from the top of the directory hierarchy downwards,
+            such that values from the top level are inherited by all data files
+            at lower levels to which it is applicable unless overridden
+            by a value for the same key present in another metadata file at a lower level
+            (though it is RECOMMENDED to minimise the extent of such overrides).
+
+        1.  There is no notion of "unsetting" a key/value pair.
+
+Corollaries:
+
+1.  As per rule 3, metadata files applicable only to a specific participant / session
+    MUST be defined in or below the directory corresponding to that participant / session;
+    similarly, a metadata file that is applicable to multiple participants / sessions
+    MUST NOT be placed within a directory corresponding to only one such participant / session.
+
+1.  It is permissible for a single metadata file to be applicable to multiple data
+    files at that level of the hierarchy or below. Where such metadata content is consistent
+    across multiple data files, it is RECOMMENDED to store metadata in this
+    way, rather than duplicating that metadata content across multiple metadata files.
+
+Example 1: Demonstration of inheritance principle
+
+{{ MACROS___make_filetree_example(
+    {
+    "sub-01": {
+        "func": {
+            "sub-01_task-rest_acq-default_bold.nii.gz": "",
+            "sub-01_task-rest_acq-longtr_bold.nii.gz": "",
+            "sub-01_task-rest_acq-longtr_bold.json": "",
+            }
+        },
+    "task-rest_bold.json": "",
+    }
+) }}
+
+Contents of file "task-rest_bold.json":
+
+```JSON
+{
+    "EchoTime": 40.0,
+    "RepetitionTime": 1000.0
+}
+```
+
+Contents of file "sub-01/func/sub-01_task-rest_acq-longtr_bold.json":
+
+```JSON
+{
+    "RepetitionTime": 3000.0
+}
+```
+
+When reading image `sub-01/func/sub-01_task-rest_acq-default_bold.nii.gz`, only
+metadata file `task-rest_bold.json` is read; file
+`sub-01/func/sub-01_task-rest_acq-longtr_bold.json` is inapplicable as it contains
+entity "`acq-longtr`" that is absent from the image path (rule 2.3). When reading image
```

Comment:
Rule 2.c, see my comment above.

I see that below this would be "5.b.ii" instead of 5.2.2, so maybe we need to discuss my suggestion - as 5.b.ii is kind of ugly.

Reply:
Revised by 8e808a12, but not yet resolving in case the enumeration formatting needs to be discussed further.

Reply:
Looking at it again, I don't find it ugly anymore :-) Thanks for making the change.